sites/website/docusaurus.config.js

```diff
@@ -3,7 +3,7 @@ module.exports = {
     tagline: "The adaptive interface system for modern web experiences",
     url: "https://www.fast.design",
     baseUrl: "/",
-    favicon: "img/favicon/favicon.ico",
+    favicon: "bundle/favicon.ico",
```

Comment:
shouldn't this be pulled from CDN?

Reply:
I've just merged my PR that includes the favicon, so yes we should now have a link to it to the effect of https://static.fast.design/assets/manifest/favicon.ico

Reply:
Thank you for letting me know. I have updated it to the CDN.